homeassistant/components/youless/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The youless integration."""
+import asyncio
+from urllib.error import URLError
+
+import voluptuous as vol
+from youless_api import YoulessAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+
+from ...exceptions import PlatformNotReady
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the youless component."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up youless from a config entry."""
+    api = YoulessAPI(entry.data[CONF_HOST])
+
+    try:
+        await hass.async_add_executor_job(api.initialize)
+    except URLError as exception:
+        raise PlatformNotReady from exception
+
+    hass.data[DOMAIN][entry.entry_id] = api
+    for component in PLATFORMS:
+        hass.async_create_task(
+            hass.config_entries.async_forward_entry_setup(entry, component)
+        )
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
```

Comment:
Return type missing.

```suggestion
async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```